app/frontend/src/pages/ask/Ask.tsx

```diff
@@ -49,16 +51,15 @@ export function Component(): JSX.Element {
     const client = useLogin ? useMsal().instance : undefined;
 
     const getConfig = async () => {
-        const token = client ? await getToken(client) : undefined;
-
-        configApi(token).then(config => {
+        configApi().then(config => {
```

Comment:
The config API does not require a user token, so I removed it. If we did merge auth setup and config API, then it would require a token at that point.

Reply:
if it's simpler to keep the separate, that's great. I was not under the impression that we had any sensitive info in the auth setup endpoint, so it's also possible I just added a auth token for no reason :)

Reply:
I looked into merging them, but they are currently called at two different times, since config is used for the inner page and authSetup is used for the navigation header (different React components). They could potentially still be merged but it'd require some re-organization of the code. This seems to work alright for now.